src/App.tsx

```diff
@@ -12,161 +13,34 @@ export const App = () => (
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
+          <NavLink
```

Comment:
You can create separate Navbar component

Reply:
> You can create separate Navbar component

Good point, create a separate component